src/main/kotlin/it/polito/wa2/group03userregistration/entities/User.kt

```diff
@@ -12,7 +13,10 @@ class User(var username: String, var password: String?, var email: String) {
 
     var salt: String = ""
 
+    var role: UserRole = UserRole.CUSTOMER
+
     @OneToOne(mappedBy = "userActivation")
     var activation: Activation? = null
 
+    var enabled = 0
```

Comment:
Yes exactly! I added it because Spring Security asks for this flag when reading users from the db to check if they are enabled to login or not. If you try to log in with a non-enabled user it should say "User not enabled" or something like that.